File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int sum = 0;
        int prod = 1;
        
        for(int i : num_list) {
            sum += i;
            prod *= i;
        }
        
        answer = (prod > sum*sum) ? 0 : 1;
        
        return answer;
    }
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.cs
using System;

public class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int a = 1;
        int b = 0;
        foreach(var i in num_list)
        {
            a *= i;
            b += i;
        }
        b *= b;
        
        return a > b ? 0 : 1;
    }
}